docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,231 +9,1402 @@
    },
```

Comment:
<p>Theres all these new words introduced like implcit dimensions, or parameter core dimensions, but theyre not listed at the top. I think its good if they were so its clear what is going to be explained throughout this doc, and provides an easy reference. If they can be linked to the heading in this notebook thatd be even better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
<p>I think that all the important terms are mentioned in the beginning, and this text clearly states we need to introduce a  new concept (core dimensions), just to understand this example. I don't think this concept is as critical to be listed at the top.</p>